dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/PublishContentActionlet.java

```diff
@@ -69,10 +74,10 @@ public void executeAction(WorkflowProcessor processor, Map<String, WorkflowActio
             }
 
         } catch (Exception e) {
+
             Logger.error(PublishContentActionlet.class, e.getMessage(), e);
```

Comment:
Please include in the error message the Identifier or useful information about the Contentlet involved in the process.